tests/test_pricing.py

```diff
@@ -0,0 +1,116 @@
+from toyaikit.pricing import PricingConfig
+from genai_prices import calc_price, Usage
+from unittest.mock import patch
+import pytest
+from io import StringIO
+import sys
+
+
+class TestPricingConfig:
+
+    def setup_method(self):
+        self.pricing_config = PricingConfig()
+
+    def test_singleton_creation(self):
```

Comment:
I don't think it's really necessary to make it a Singleton. I'd remove this test and the logic for keeping it a singleton